### DISCUSSION TITLE
Page title as file name?

### DISCUSSION THREAD

**Discussion by reisuart** [Frederisk/Wikitext-VSCode-Extension#56] [Ideas]
Hello, so i kinda work with multiple pages at once, and having the file name set with the page title would be really helpful tbh! (though since subpages use / a nice idea would be to use - or other punctuation to keep the disctinction for subpages)

**Comment by Frederisk** [comment]
I have had this idea a long time ago, but I didn't find a property or method to control the title in [`TextDocument`](https://code.visualstudio.com/api/references/vscode-api#TextDocument).🤔

**Reply by reisuart** [reply]
That's sad! I hope you'll find a way somehow!
